src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -2558,6 +2558,35 @@ namespace wallet_rpc
     typedef epee::misc_utils::struct_init<response_t> response;
   };
 
+  struct COMMAND_RPC_GET_MULTISIG_KEY_EXCHANGE_BOOSTER
+  {
+    struct request_t
+    {
+      std::string password;
+      std::vector<std::string> multisig_info;
```

Comment:
The `multisig_info` here is a vector of `b58` strings, because we want the multisig stuff to be copy-pastable.

Reply:
Ah crap, I forgot about that. I think I even mentioned this to you?

Reply:
I don't remember, but I do remember stuff breaking when I didn't use `b58` (maybe the deserializer can't handle a vector of strings that don't contain characters? I don't remember exactly).

Reply:
Oh yeah, I remember the discussion more clearly now - we definitely selected base58 because it also helped with passing messages via email, etc. Damn, too focused on serialization or something :/